docs/reference/ilm/actions/ilm-rollover.asciidoc

```diff
@@ -7,6 +7,13 @@ Phases allowed: hot.
 Rolls over a target to a new index when the existing index satisfies
 the specified rollover conditions.
 
+[NOTE]
+====
+When an index is rolled over, a new index is created with a `creation_date` that reflects
+the rollover time. This impacts the calculation of downstream {ilm} `max_age` and `min_age` conditions. 
+<<min-age-calculation,Learn more>>.
```

Comment:
I think the "This" in the second sentence is ambiguous. For instance, after an index is rolled over, there is no "downstream ILM `max_age`" condition, the only time comparison is for the `min_age` of phase definitions.

I think we could clarify this to something like:
```suggestion
When an index is rolled over, a new index is created with a new `creation_date` while the previous index
is updated with a timestamp that reflects the rollover time. This rollover time impacts the calculation of downstream {ilm} `min_age` phase calculations. 
<<min-age-calculation,Learn more>>.
```

(feel free to change this, I just put it here as a suggestion)

Reply:
> isn't it the `creation_date` that's used in `min_age` phase calculation?

No, ILM uses the rollover time (the time at which the index was rolled over). In your suggestion it makes it sound as though the new index has a `creation_date` which then gets used for ILM `min_age` phase calculations, when it's really the _old_ write index that has a new "time" set (the rollover time) that is used for the `min_age` calculations. The newly created index is back at the beginning again, so it's likely back to the `rollover` step.

For example, if a user has a policy where the `hot` phase is configured with a rollover action for `max_docs: 1`, and a `warm` phase with a `min_age: 1h`. Say it takes 10 minutes before a single document is indexed (to meet the `max_docs: 1` condition), so it rolls over at the 10 minute mark. The index records the time it rolls over, then waits an additional 30 minutes from this time (for a total of 40 minutes) before entering the `warm` phase.